bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -85,9 +85,11 @@ <h2 class="section-title"><span>{{ _('Common Voice') }}</span></h2>
 
         <div class="card-content-wrapper">
           <div class="card-content">
-            <a href="https://voice.mozilla.org/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub" data-link-name="Record in Common Voice" data-link-type="link" data-link-position="common-voice">
-              <h3 class="card-heading">{{ _('Let your voice be heard… literally') }}</h3>
-            </a>
+            {% trans url='https://voice.mozilla.cz/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub' if LANG == 'cs' else 'https://voice.mozilla.org/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub' %}
+              <a href="{{ url }}" data-link-name="Record in Common Voice" data-link-type="link" data-link-position="common-voice">
+                <h3 class="card-heading">{{ _('Let your voice be heard… literally') }}</h3>
```

Comment:
This is causing the page to throw a 500 error, since you're nesting a translateble string inside a `trans` block. Please use the `set` method here too (and test your changes locally, to make sure it's working). Thanks